src/main/resources/templates/payload/NEW_ADDRESS_REPORTED-event.ftl

```diff
@@ -38,8 +38,10 @@
       "longitude" : "${address.longitude?string["0.#########"]}",
     </#if>
       "region" : "${region}",
+    <#if sourceCase != "NEW_SPLIT_ADDRESS">
```

Comment:
check https://collaborate2.ons.gov.uk/confluence/display/CATD/FWMT+Interfaces+-+Outcomes+-+RM+-+New+Split+Address

Reply:
What are you referring too? region? cause region is removed in a bug fix